src/move/econia/sources/user.move

```diff
@@ -2240,7 +2243,9 @@ module econia::user {
                 &mut market_account_ref_mut.quote_ceiling
             ) else abort E_ASSET_NOT_IN_PAIR;
         // Assert enough asset available for withdraw.
-        assert!(amount <= *available_ref_mut, E_WITHDRAW_TOO_LITTLE_AVAILABLE);
+        if (amount > *available_ref_mut) {
+            amount = *available_ref_mut
+        };
```

Comment:
I believe this can result in an accounting error when routing assets between counterparties.